### PR TITLE
Add struct for mailing sources #40

### PR DESCRIPTION
**Pull request by usox**

### CODE REVIEW

File: tests/Struct/MailingTemplate/MailingTemplatesSourcesTest.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scn\EvalancheSoapStruct\Struct\MailingTemplate;
+
+use PHPUnit\Framework\TestCase;
+
+class MailingTemplatesSourcesTest extends TestCase
```
Comment (WorksDev) [inline tests/Struct/MailingTemplate/MailingTemplatesSourcesTest.php]: Please use StructTestCase

File: tests/Struct/MailingTemplate/MailingTemplatesSourcesTest.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    public function setUp(): void
+    {
+        $this->subject = new MailingTemplatesSources();
+    }
+
+    /**
+     * @dataProvider methodDataProvider
+     */
+    public function testSetterAndGetter(
```
Comment (WorksDev) [inline tests/Struct/MailingTemplate/MailingTemplatesSourcesTest.php]: Is it possible to move the Method into StructTestCase

File: src/Struct/MailingTemplate/MailingTemplatesSources.php [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+namespace Scn\EvalancheSoapStruct\Struct\MailingTemplate;
+
+/**
+ * Contains the source code of the mailing views
+ */
+class MailingTemplatesSources implements MailingTemplatesSourcesInterface
+{
+
+    /** @var string */
+    private $template_email;
```
Comment (WorksDev) [inline src/Struct/MailingTemplate/MailingTemplatesSources.php]: according to the other Classes, Variables should be lower- cc maybe u can use php-cs-fixer to change that

**Review by WorksDev — CHANGES_REQUESTED** [review]

File: tests/Struct/MailingTemplate/MailingTemplatesSourcesTest.php (hunk 2 above)
Comment (usox) [reply tests/Struct/MailingTemplate/MailingTemplatesSourcesTest.php]: Sure I can, but this will require a separate testcase

**Review by WorksDev — APPROVED** [review]

**Pull request merged**